script.js

```diff
@@ -0,0 +1,55 @@
+class Ship {
+  constructor(itinerary) {
+    this.itinerary = itinerary;
+    this.currentPort = itinerary.ports[0];
+    this.previousPort = null;
+    this.remainingPort = [...itinerary.ports];
+    this.currentPort.addShip(this);
+  }
+  setSail() {
```

Comment:
Its good practice to keep a clear line between each of your methods as it makes things easier to read